core/src/main/java/com/aem/demo/core/dto/NewsCard.java

```diff
@@ -0,0 +1,21 @@
+package com.aem.demo.core.dto;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Date;
+
+@Getter
```

Comment:
(cosmetic) You can use `@Data` which is @Getter + @Setter
But do you actually need to _set_ values of a data model?)